api/v1beta1/openstackdataplanenode_types.go

```diff
@@ -38,6 +39,10 @@ type OpenStackDataPlaneNodeSpec struct {
 	// +kubebuilder:validation:Optional
 	// AnsibleHost SSH host for Ansible connection
 	AnsibleHost string `json:"ansibleHost,omitempty"`
+
+	// +kubebuilder:default=true
+	// Deploy boolean to trigger ansible execution
+	Deploy bool `json:"deploy"`
```

Comment:
What is the purpose of this? Dev/testing?

Reply:
Partially, yes. The nova-operator is also adding something similar. We may eventually need something finer grained, but for the time being it also allows for orchestration between baremetal and the ansible, in that you might just want to deploy baremetal, and then stop. Then deploy the ansible at a later time.